lms/static/sass/inline-discussion-rtl.scss

```diff
@@ -0,0 +1,23 @@
+// From lms-main-v1-rtl.scss
```

Comment:
@andy-armstrong Yup, that makes sense. This commit addresses your comment: https://github.com/edx/edx-platform/pull/13483/commits/524599ea2e469fc90a14ec71ac29c4d7a99ad328.